### ISSUE TITLE
Parsing `check` from `makepkg.conf`

### ISSUE THREAD

**Issue by fosskers** [fosskers/aura#793]
`check` is on by default. It should be parsed for, and if there, should make Aura also consider all `checkdeps` within `dependencies.rs`.

Otherwise, this is preventing certain packages from being built.

**Comment by fosskers** [comment]
While this field still isn't parsed from `makepkg.conf`, a `--nocheck` flag has been added.

**Comment by ultimatespirit** [comment]
Was this `--nocheck` flag only added to the Rust version of aura, or is it in the AUR version as well, and we should be using the `aura-git` package? Came here to report this exact bug as I've had to just use the hot-editing workflow to get around this till now.

Relatedly, I just noticed that my install of the `aura` AUR package has not been updated in about two years now. Should we just consider aura out of date / deprecated until the rust version gets added to the AUR? I've more and more often had to just use `pkgctl` to get around issues in `aura`, which has been a little sad.

Normally I'd just edit the AUR PKGBLD to make a version that tracks the rust version, but due to aura's lack of a blacklisting feature (unless that does exist now?) I'm pretty sure it'd end up re-installing itself from the AUR immediately and downgrade itself basically... So I never bothered doing that haha.
